addons/binding/org.openhab.binding.max/ESH-INF/thing/bridge.xml

```diff
@@ -86,8 +87,23 @@
 				<default>1000</default>
 				<advanced>true</advanced>
 			</parameter>
+
+			<!-- Trigger inclusion mode action from habmin menu -->
+			<parameter name="action-cubeInclusion" type="integer" groupName="actions">
```

Comment:
Wouldn't it be much better to have inclusion done through discovery?
I.e. background discovery simply reads what is available on the cube, but when actively triggering a discovery, the cube should go in inclusion mode. I think this is how it is done for ZWave as well and also [the hue reference implementation](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueLightDiscoveryService.java#L78).